### PR TITLE
Added Required FIles

### PR DESCRIPTION
**Pull request by vr-varad**
Added
- Contributing.md
- Code_of_conduct.md
- basic issue tempate
@shamimbinnur review it whenever u are free.

### CODE REVIEW

**Comment by vr-varad** [comment]
resolves #5

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Code of Conduct - zone-out
```
Comment (shamimbinnur) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
# Code of Conduct - ZoneOut
```

**Comment by shamimbinnur** [comment]
@vr-varad  great work! Thanks! There is a simple typo that should be fixed. Also, for now, could you please not include the issue-template in this PR? We will work on it later with a different PR.

**Comment by vr-varad** [comment]
Sure. @shamimbinnur

**Comment by shamimbinnur** [comment]
> Sure. @shamimbinnur

Also, It would be much appreciated if you could link the `Contributing.md` and `Code_of_conduct.md` files to the `readme.md`, thanks @vr-varad

**Comment by vr-varad** [comment]
@shamimbinnur resolved the suggested errors and add them to readme.

**Comment by vr-varad** [comment]
@shamimbinnur just let me know if there are any more issues to resolve. ;)

**Pull request merged**